common/tests/threadpool_int/threadpool_int.c

```diff
@@ -409,7 +410,7 @@ TEST_FUNCTION(MU_C3(scheduling_, N_WORK_ITEMS, _work_with_pool_threads))
     execution_engine_dec_ref(execution_engine);
 }
 
-#define N_THREADPOOL_TIMERS 100
+#define N_THREADPOOL_TIMERS 1
 
 TEST_FUNCTION(MU_C3(starting_, N_THREADPOOL_TIMERS, _timer_start_runs_once))
```

Comment:
>starting_, N_THREADPOOL_TIMERS, _timer_start_runs_once) [](http://example.com/codeflow?start=20&length=55)

I know you didn't do this, but it seems that this name is misleading, it should just say something about 1 timer to run once.

Reply:
done